docs/src/operations/reference/manipulation/make-contiguous.rst

```diff
@@ -0,0 +1,6 @@
+make_contiguous
+===============
+
```

Comment:
Maybe we add here the links to specific functions in numpy and torch

```suggestion

These operations are equivalent of :py:func:`np.ascontiguousarray(array) <numpy.ascontiguousarray>` in Numpy and :py:func:`tensor.contiguous() <torch.Tensor.contiguous>` in torch.
```

Reply:
done!